src/Morphir/Elm/IncrementalFrontend.elm

```diff
@@ -577,9 +577,16 @@ processModule moduleName parsedModule opts exposedModules repo =
 
                     else
                         Private
+
+                documentedModule : Module.Definition () (Type ())
```

Comment:
Could you just use an emptyDefinition here instead of creating a new module which is empty anyway.

Reply:
It's not a completely empty module, the doc field is set.

Reply:
It's not completely empty, it has docs.

Reply:
Ok. This is fine.